content/nginxaas-azure/overview/overview.md

```diff
@@ -44,6 +56,10 @@ NGINXaaS for Azure is supported in the following regions:
 
 - Azure management tools (API, CLI, portal, terraform) work with NGINXaaS to create, update, and delete deployments
 - Each NGINXaaS deployment has dedicated network and compute resources. There is no possibility of [noisy neighbor problems](https://learn.microsoft.com/en-us/azure/architecture/antipatterns/noisy-neighbor/noisy-neighbor) or data leakage between deployments
+- NGINXaaS can load balance even if upstream servers are located in different geography as long as no networking limitations are mentioned in the [Known Issues]({{< ref "/nginxaas-azure/known-issues.md" >}})
+- NGINXaaS supports request tracing. See the [Application Performance Management with NGINX Variables](https://www.f5.com/company/blog/nginx/application-tracing-nginx-plus) blog to learn more about tracing.
+- Supports HTTP to HTTPS, HTTPS to HTTP, and HTTP to HTTP redirects. NGINXaaS also provides the ability to create new rules for redirecting. See [How to Create NGINX Rewrite Rules | NGINX](https://www.nginx.com/blog/creating-nginx-rewrite-rules/) for more details.
+- NGINXaaS supports VNet, and VPN gateways if they do not have limitations. Known limitations can be found in the [Known Issues]({{< ref "/nginxaas-azure/known-issues.md" >}}).
```

Comment:
```suggestion
- NGINXaaS supports VNet, and VPN gateways if they do not have limitations. Known limitations can be found in the [Known Issues]({{< ref "/nginxaas-azure/known-issues.md" >}}).
```

Have tried out VPN gateways at all? if not, maybe redact this until we do? I remember we had issues with epxressroutes once so calling out non-test technologies may not be a good idea.

Reply:
This is in the FAQ in the production docs:
https://docs.nginx.com/nginxaas/azure/faq/#:~:text=What%20are%20the,Known%20Issues.

Reply:
The scope of this PR is to move the content from the FAQ to relevant docs. We can:

- Move it somewhere else more relevant.
- Leave it as is in this PR and work on improving it separately.
- Delete it.

Reply:
```suggestion
- NGINXaaS is deployed inside of your Azure network and can connect to your upstream application running in your ecosystem. Known networking limitations can be found in the [Known Issues]({{< ref "/nginxaas-azure/known-issues.md" >}}).
```

Reply:
👍